src/components/form-def/form-def.component.js

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import TextArea from "../text-area/text-area.component";
+import { SelectFood } from "../select/select.component";
+import BtnContainer from "../btn-container/btn-container.component";
+
+const DefaultForm = ({ tableContent, setTableContent }) => {
+  const [state, setState] = useState({ day:'', meal:'', boxContent:'', alert:false })
```

Comment:
You only update `alert` when you submit but it's a different case for the meal, day and content so they serve different purposes and it's better to separate them.
```suggestion
  const [form, updateForm] = useState({ day:'', meal:'', boxContent:'' })
  const [alert, setAlert] = useState(false)
```